package.json

```diff
@@ -9,37 +9,39 @@
     "url": "http://github.com/insin/react-hn.git"
   },
   "scripts": {
-    "build": "npm run lint && cp node_modules/sw-toolbox/build/sw-toolbox.js public/sw-toolbox.js && ./node_modules/.bin/react build src/index.js public/build && npm run precache",
+    "build": "npm run lint && cp node_modules/sw-toolbox/sw-toolbox.js public/sw-toolbox.js && react build src/index.js public/build && npm run precache",
     "deploy": "gcloud preview app deploy",
-    "lint": "./node_modules/eslint-config-jonnybuchanan/bin/lint.js src",
-    "lint:fix": "./node_modules/eslint-config-jonnybuchanan/bin/lint.js --fix .",
+    "lint": "eslint src server.js",
+    "lint:fix": "npm run lint -- --fix",
     "start": "node server.js",
     "postinstall": "npm run build",
-    "serve": "./node_modules/.bin/nwb serve",
-    "precache": "./node_modules/sw-precache/cli.js --root=public --config=sw-precache-config.json"
+    "serve": "nwb serve",
+    "precache": "sw-precache --root=public --config=sw-precache-config.json"
   },
```

Comment:
I think I may have needed to use these paths to get the Node on GAE deploys happy again. I'll try it out with these changes and see if we run into any trouble at all.